### ISSUE TITLE
[question] Devise JWT alongside standard Devise sign in?

### ISSUE THREAD

**Issue by mike-oodle** [waiting-for-dev/devise-jwt#93]
Hello!

I've used this gem for a pure API, however I have new circumstances where I need both a JWT and standard Devise sign in behavior.  I reached for this gem but I have run into a few snags and I just want to make sure this is:

1) Possible
2) Encouraged

Thanks for all your work and I look forward to hearing back about whether API sign in and standard Devise sign in are able to live alongside each other in the same Rails app.

**Comment by waiting-for-dev** [comment]
Yes, it is possible. Just be sure to use and understand [request_formats](https://github.com/waiting-for-dev/devise-jwt#request_formats) and the [session storage caveat](https://github.com/waiting-for-dev/devise-jwt#session-storage-caveat).

**Comment by waiting-for-dev** [comment]
Feel free to ask if something is not clear :+1:
